crates/biome_js_analyze/tests/specs/complexity/noUselessLabel/invalid.jsonc.snap

```diff
@@ -371,8 +371,9 @@ invalid.jsonc:1:27 lint/complexity/noUselessLabel  FIXABLE  ━━━━━━
   i Safe fix: Remove the unnecessary label.
     You can achieve the same result without the label.
   
-    1 │ A:·while(true)·{·continue·A/*after*/;·}
-      │                          --            
+  - A:·while(true)·{·continue·A/*after*/;·}
+  + A:·while(true)·{·continue/*after*/;·}
```

Comment:
Line number seems to be missing when it is `1`.

Reply:
I think this is also an existing issue. Like this [one](https://github.com/biomejs/biome/pull/2237/files/f22ffa7904a2f2cbfc34fb370eca32197209a86a#diff-b13055313a857d44aa05d85cdad155b0a0b61005de8f728e578c66fcae8af6dc), the existing diff doesn't have  the line number.